lib/ronin/web/browser/agent.rb

```diff
@@ -106,7 +106,7 @@ def self.open(**kwargs)
             browser.quit
           end
 
-          return browser
+          browser
```

Comment:
I'm a fan of sometimes using explicit `return`s. I feel it's weird to have a local variable just sitting there on the last line. I know, it's a weird habit that I developed after years of writing C/C++/Java...